### DISCUSSION TITLE
Joomla versão 4.2 install

### DISCUSSION THREAD

**Discussion by cosmoelton** [joomla/joomla-cms#39319] [Show and tell]
Hello, I'm new here, and with joomla too, I'm having a lot of trouble installing Joomla, would anyone have a step by step?

**Comment by ghost** [comment]
Hi,

All documents: https://docs.joomla.org/Main_Page

**Comment by cosmoelton** [comment]
Joomlaphp,

llo, I'm looking for something more explanatory, because I'm trying to implement joomla without success.

I already configured apache2 - OK
PHP 7.4.33-OK
Create Database -OK
I already gave permission on the folder where the joomla files are

**Comment by drmenzelit** [comment]
Sorry for the very late answer, but this kind of questions are better placed in the forum: forum.joomla.org
